core/src/main/java/com/scalar/db/api/Admin.java

```diff
@@ -417,6 +418,30 @@ void repairTable(
   void addNewColumnToTable(String namespace, String table, String columnName, DataType columnType)
       throws ExecutionException;
 
+  /**
+   * Adds a new column to an existing table. The new column cannot be a partition or clustering key.
+   * <br>
+   * See {@link #addNewColumnToTable(String, String, String, DataType)} for more information.
+   *
+   * @param namespace the table namespace
+   * @param table the table name
+   * @param columnName the name of the new column
+   * @param columnType the type of the new column
+   * @param encrypted whether the new column should be encrypted
+   * @throws IllegalArgumentException if the table does not exist or the column already exists
+   * @throws ExecutionException if the operation fails
+   */
+  default void addNewColumnToTable(
+      String namespace, String table, String columnName, DataType columnType, boolean encrypted)
+      throws ExecutionException {
+    if (encrypted) {
+      throw new UnsupportedOperationException(
+          CoreError.ENCRYPTED_COLUMNS_NOT_SUPPORTED.buildMessage());
+    } else {
+      addNewColumnToTable(namespace, table, columnName, columnType);
+    }
+  }
```

Comment:
@feeblefakie @komamitsu @Torch3333 Sorry, I've also added this method. Could you please take a look when you have time? 🙇